src/targets/mas_steekproef_zavelberg/_targets_r/targets/hulpvariabelen.R

```diff
@@ -2,15 +2,13 @@ list(
   tar_target(
     name = plus_sb,
     command = add_bo_to_frame(
-      punten_df = punten_steekproefkader,
```

Comment:
Bij dit vraag ik me af of er toch geen pattern = map() mag blijven staan. Momenteel is het enkel de zandleemstreek, maar als de we deze pipeline recycleren voor meerdere landbouwstreken zal er ergens een mapping over landbouwstreken moeten overblijven. Voor nu maakt het echter niets uit.

Reply:
Well spotted. I will open a new branch for this.